polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -12,7 +12,6 @@ $breakpoint-small: 458px;
   --pc-index-table-loading-panel: 37;
   position: relative;
   overflow: hidden;
-  border-radius: inherit;
```

Comment:
I'm a fan of removing this due to it's dependence on the parent style, but looks like it is needed for rounding the corner when the IndexTable is wrapped in a Card.

<img width="193" alt="12-05-s5r0v-1w62u" src="https://user-images.githubusercontent.com/11774595/184371388-3d903726-16f6-4b66-8497-ca5da89884e7.png">